homeassistant/components/proxmoxve/config_flow.py

```diff
@@ -0,0 +1,169 @@
+"""Config Flow for ProxmoxVE."""
+import logging
+
+import proxmoxer
+from requests.exceptions import ConnectTimeout, SSLError
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_PASSWORD,
+    CONF_PORT,
+    CONF_USERNAME,
+    CONF_VERIFY_SSL,
+)
+
+from . import ProxmoxClient
+from .const import CONF_REALM, DEFAULT_PORT, DEFAULT_REALM, DEFAULT_VERIFY_SSL, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ProxmoxVEConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """ProxmoxVE Config Flow class."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
+
+    def __init__(self):
+        """Init for ProxmoxVE config flow."""
+        super().__init__()
+
+        self._config = {}
+        self._proxmox_client = None  # type: ProxmoxClient
+
+    async def async_step_import(self, import_config=None):
+        """Import existing configuration."""
+        _LOGGER.info("Importing ProxmoxVE config")
+
+        return await self.async_step_init(import_config, True)
+
+    # pylint: disable=signature-differs
+    async def async_step_user(self, user_input):
+        """Manual user configuration."""
+        return await self.async_step_init(user_input, False)
+
+    # pylint: disable=signature-differs
```

Comment:
We should avoid disabling Pylint for these reasons. Instead, e.g., an instance variable can be used to make that difference.